analytics/views/remote_activity.py

```diff
@@ -72,14 +72,20 @@ def get_remote_server_activity(request: HttpRequest) -> HttpResponse:
         "Links",
     ]
 
+    # If the column order above changes, update the constants below
+    SERVER_ID = 0
+    SERVER_HOSTNAME = 1
+    LAST_AUDIT_LOG_DATE = 4
+    MOBILE_USER_COUNT = 5
+    MOBILE_PUSH_COUNT = 6
```

Comment:
This helped me with the different iterative changes on the for loops below, but there might be a better method for this. This also means that the column strings can be changed without breaking any logic below.

Reply:
Yeah this seems better than what we had before!